### DISCUSSION TITLE
failed push to Synology

### DISCUSSION THREAD

**Discussion by peoute** [reddec/syno-cli#2] [Q&A] [answered]
I think I made it through most of the set up and I see the cert in the folder on the NAS, but the certificate doesn't show up in the certificate tab under security in DSM.

In the log it looks like this is the error. I tried updating the url in the environment variables a few different ways, but that didn't seem to change this.

`failed push to Synology: login: invoke api: get API SYNO.API.Auth version: invoke api: execute request: Post "http://localhost:5000/webapi/query.cgi?": dial tcp 127.0.0.1:5000: connect: connection refused`

Any thoughts?

**Comment by reddec** [comment, marked as answer]
Hi! Seems like you didn't set up `SYNOLOGY_URL` as defined in Environment section in the article

**Reply by peoute** [reply]
Ah, sorry. I thought that was the issue and was trying different addresses for that variable, but the issue was somehow the variable name was just "URL" instead of "SYNOLOGY_URL." I thought that was one of the default variable names when I loaded the container, so not sure how it changed, but it seems like all is well! Thanks!
